pkg/test/test_data/patch/01-assert.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: toy1-nginx-deployment
+  labels:
+    app: "toy"
+    version: "1.0.0"
+    instance: "toy1"
+  annotations:
+    plan: "scale"
+    step: "scale"
+    phase: "scale"
+spec:
+  replicas: 3
+  template:
```

Comment:
If you want, you can drop the rest of the template here, and rewrite this assert:

```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: toy1-nginx-deployment
spec:
  replicas: 3
```

IMO it makes it more clear what is being tested and more DRY

Reply:
Would this still validate that the annotations change from `deploy` to `scale`, but the labels do not?

Reply:
My example? No. But you can add that too:

```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: toy1-nginx-deployment
  labels:
    app: "toy"
    version: "1.0.0"
    instance: "toy1"
  annotations:
    plan: "scale"
    step: "scale"
    phase: "scale"
spec:
  replicas: 3
```

The asserts let you be as specific or general as you need.